services/decoder/protocols/connext/constants/domain-mapping.ts

```diff
@@ -0,0 +1,11 @@
+export const DOMAIN_ID_TO_CHAIN_ID: { [domain_id: number]: string } = {
+    6648936: "Ethereum Mainnet",
+    1886350457: "Polygon",
+    1869640809: "Optimism",
+    1634886255: "Arbitrum One",
+    6778479: "Gnosis Chain",
+    6450786: "BNB Chain",
+    1818848877: "Linea",
+    1835365481: "Metis",
+    1650553709: "Base",
+};
```

Comment:
No need for a separate file, create this in the decoder file itself

Reply:
Resolved in: 44b5616f9a2fc157809c3ca616cdcfa3d50ef712